docs/vault/upgrading.md

```diff
@@ -120,3 +120,100 @@ sudo systemctl start kintsugi-vault.service
 <!-- tabs:end -->
 
 OR start the [process manually](vault/installation?id=_5-start-the-vault-client).
+
+## Vault Auto-Updater
```

Comment:
I think the auto-updater should be part of the installation procedure.
1. Insert a new "Auto-Upgrading Installation" above the https://docs.interlay.io/#/vault/installation?id=standard-installation 
1. Keep the https://docs.interlay.io/#/vault/installation?id=standard-installation

Reply:
The "Standard Installation" section includes details on how to install and run the bitcoin node. I will move those to the section on the auto-upgrading installation and only reference in them in the standard installation.